CenterForOpenScience/osf.io
# [feature] Make retracted registration supplemental pages error [OSF-7978]

## Purpose
Retracted registrations allow users to go to the the supplemental pages (e.g., wikis, files, etc) with varying degrees of success (most have errors and bad formatting, missing contributors, etc).


## Changes
Make supplemental pages for retracted registrations redirect back to the main retraction page.



## Side effects
1. Despite being missing from the nav bar on registrations, you can still navigate to the /registrations/ route, and click the 'register' button which ends up having an error. I made that page error out as well for all registrations.
2. Navigating to a fake registration schema route lead to an unable to resolve. It should now lead to a "No registration schema with that name could be found." error



## Ticket

https://openscience.atlassian.net/browse/OSF-7978

## QA Notes
1. Test that you cannot go to the /registrations/ route on a non- withdrawn registration
2. Test that you get redirected to a working retraction page when you navigate to any of these routes for a retracted registration:
files, wiki/home, analytics, forks, contributors, settings, withdraw, register, 'register/<schema_id>/
3. Test that those routes work properly for normal registrations and nodes
4. Confirm that you get a proper error message for /register/<fake_schema_id>/ for a bad schema and that proper schemas still work as expected
